vitrina/datasets/migrations/0004_auto_20221010_1419.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.16 on 2022-10-10 11:19
+
+from django.db import migrations
+
+
+def create_history_objects(apps, schema_editor):
+    DatasetEvent = apps.get_model('vitrina_datasets', 'DatasetEvent')
+    Dataset = apps.get_model('vitrina_datasets', 'Dataset')
+    Revision = apps.get_model('reversion', 'Revision')
+    Version = apps.get_model('reversion', 'Version')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    for event in DatasetEvent.objects.exclude(dataset_id__isnull=True)[:10]:
```

Comment:
```suggestion
    for event in DatasetEvent.objects.exclude(dataset_id__isnull=True):
```

And could you please remove this, I left it accidentally.